x/observer/simulation/operation_disable_fast_confirmation.go

```diff
@@ -0,0 +1,82 @@
+package simulation
+
+import (
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	zetasimulation "github.com/zeta-chain/node/testutil/simulation"
+	"github.com/zeta-chain/node/x/observer/keeper"
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+// SimulateDisableFastConfirmation generates a MsgDisableFastConfirmation and delivers it.
```

Comment:
Note for future PR: no need to add simulation for new messages

Reply:
Noted.